pep-0554.rst

```diff
@@ -973,7 +973,7 @@ following:
 
 * importing an extension module fails if it does not implement the
   PEP 489 API
-* new threads are not allowed (including daemon threads)
```

Comment:
The reminder about daemon threads is with keeping, I think.  How about shifting it around to resolve the clarity problem: "new threads (including daemon threads) are not allowed"?